ios/browser/api/local_data_file_service/local_data_file_service_wrapper+private.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_LOCAL_DATA_FILE_SERVICE_LOCAL_DATA_FILE_SERVICE_WRAPPER_PRIVATE_H_
+#define BRAVE_IOS_BROWSER_API_LOCAL_DATA_FILE_SERVICE_LOCAL_DATA_FILE_SERVICE_WRAPPER_PRIVATE_H_
+
+#include "brave/ios/browser/api/local_data_file_service/local_data_file_service_wrapper.h"
+
+/// A private extension for `LocalDataFileServiceWrapper`
+/// which is usable inside the brave core library and not exposed to iOS.
+@interface LocalDataFileServiceWrapper (Private)
+/// Start the service so that it begins downloading/updating the necessary
+/// components.
+- (void)start;
```

Comment:
If this is the only thing wrapped and available from the data service then it doesn't need to be a wrapper at all, all of its code can live in `ios/browser/local_data_file_service` and be used privately.

Reply:
Not quite sure what you mean. this is a +private.h header. there are a number of files stored on this class and this is just the private header method available.

Reply:
I mean that you've conflated the URL sanitizer with the local data file service which are separate things. The only API you actually need exposed is the url sanitizer one to get a clean URL, nothing from the file service needs to be wrapped or exposed. You could just set it up with the C++ parts from BraveCoreMain

Reply:
This should no longer apply as there is no more local data file service wrapper